tests/conftest.py

```diff
@@ -9,6 +10,14 @@
 from app.config import RuntimeEnvironment
 
 
+@pytest.fixture(scope="session", autouse=True)
+def database_name():
+    patch = MonkeyPatch()
+    patch.setenv("INVENTORY_DB_NAME", "insights_test")
+    yield patch
+    patch.undo()
```

Comment:
We can use our [_set_environment_](https://github.com/rodrigonull/insights-host-inventory/blob/304d26d3ab045c82f839b9506dd7635ce7d6c83a/tests/test_utils.py#L7) utility function. It’s not the best one out there, but we use it already in some tests. It’d be at least consistent.

```suggestion
    from tests.test_utils import set_environment
    with set_environment({"INVENTORY_DB_NAME": "insights_test"}):
       yield
```